recipes/gmwi2/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
you could move this one -liner into the meta.yaml, see other packages

Reply:
Got it, I have implemented this, thanks